### PR TITLE
Update for Phoenix 1.4 

### PR DESCRIPTION
**Pull request by willfore**
Updated documentation to properly reflect implementing in a Phoenix 1.4 application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,11 +28,13 @@ config :simple_auth,
   user_session_api: SimpleAuth.UserSession.HTTPSession # See Advanced section for more options
 ```
 
-### Create a user model
-
+### Create a user context
```
Comment (chazsconi) [inline README.md]: Actually we are creating a user schema here, within the `Accounts` context.

File: README.md [hunk 2]
```diff
@@ -28,11 +28,13 @@ config :simple_auth,
...
+In this example we are using an Account context
```
Comment (chazsconi) [inline README.md]: In the example it's `Accounts` not `Account` that you have

File: README.md [hunk 3]
```diff
@@ -28,11 +28,13 @@ config :simple_auth,
...
 ```elixir
-defmodule MyProject.User do
-  use MyProject.Web, :model
+defmodule MyProject.Accounts.User do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias MyProject.Accounts.User
```
Comment (chazsconi) [inline README.md]: maybe better to put `alias __MODULE__`

File: README.md [hunk 4]
```diff
@@ -326,8 +328,8 @@ config :exldap, :settings,
   ssl: false
 ```
 
-### User model and migrations differences
-Create a user model and migrations (as above) but only include the `username`, `roles` and timestamp columns.
+### User context and migrations differences
```
Comment (chazsconi) [inline README.md]: "User schema and migrations differences"

**Review by chazsconi — COMMENTED** [review]
Thanks for this - just a few minor comments

File: README.md [hunk 5]
```diff
@@ -12,13 +12,13 @@ Add `simple_auth` to your list of dependencies in `mix.exs`:
 
 ```elixir
 def deps do
-  [{:simple_auth, "~> 1.6.0"}]
+  [{:simple_auth, "~> 1.8.0"}]
```
Comment (chazsconi) [inline README.md]: Wasn't this in the previous PR?

**Comment by willfore** [comment]
Ooops, I missed that verbiage. Yes, it was I forgot to pull the master
first. Sorry about that.

**Comment by willfore** [comment]
Should be good to merge now. :)

**Comment by chazsconi** [comment]
Thanks for the changes - did you rebase against master as the version change is still there?  Also there's another comments about `Account` not `Accounts` that is still unresolved.

**Comment by chazsconi** [comment]
Hi @willfore just checking that you saw my last comment?

**Comment by chazsconi** [comment]
@willfore Thanks a lot!

**Pull request merged**